packages/@aws-cdk/pipelines/package.json

```diff
@@ -32,46 +32,55 @@
     "organization": true
   },
   "devDependencies": {
+    "@aws-cdk/assert-internal": "0.0.0",
+    "@aws-cdk/aws-apigateway": "0.0.0",
+    "@aws-cdk/aws-ecr-assets": "0.0.0",
+    "@aws-cdk/aws-s3": "0.0.0",
+    "@aws-cdk/aws-sns-subscriptions": "0.0.0",
     "@types/jest": "^26.0.23",
     "cdk-build-tools": "0.0.0",
     "cdk-integ-tools": "0.0.0",
     "cfn2ts": "0.0.0",
-    "pkglint": "0.0.0",
-    "@aws-cdk/aws-s3": "0.0.0",
-    "@aws-cdk/aws-ecr-assets": "0.0.0",
-    "@aws-cdk/assert-internal": "0.0.0"
+    "pkglint": "0.0.0"
   },
   "peerDependencies": {
-    "constructs": "^3.3.69",
-    "@aws-cdk/core": "0.0.0",
     "@aws-cdk/aws-codebuild": "0.0.0",
     "@aws-cdk/aws-codepipeline": "0.0.0",
     "@aws-cdk/aws-codepipeline-actions": "0.0.0",
+    "@aws-cdk/aws-ec2": "0.0.0",
     "@aws-cdk/aws-ecr": "0.0.0",
     "@aws-cdk/aws-events": "0.0.0",
     "@aws-cdk/aws-iam": "0.0.0",
-    "@aws-cdk/aws-ec2": "0.0.0",
+    "@aws-cdk/aws-lambda": "0.0.0",
+    "@aws-cdk/aws-s3-assets": "0.0.0",
     "@aws-cdk/aws-secretsmanager": "0.0.0",
+    "@aws-cdk/aws-sns": "0.0.0",
     "@aws-cdk/cloud-assembly-schema": "0.0.0",
-    "@aws-cdk/aws-s3-assets": "0.0.0",
-    "@aws-cdk/cx-api": "0.0.0"
+    "@aws-cdk/core": "0.0.0",
+    "@aws-cdk/cx-api": "0.0.0",
+    "constructs": "^3.3.69"
   },
   "dependencies": {
-    "constructs": "^3.3.69",
-    "@aws-cdk/core": "0.0.0",
     "@aws-cdk/aws-codebuild": "0.0.0",
     "@aws-cdk/aws-codepipeline": "0.0.0",
     "@aws-cdk/aws-codepipeline-actions": "0.0.0",
-    "@aws-cdk/cloud-assembly-schema": "0.0.0",
+    "@aws-cdk/aws-ec2": "0.0.0",
     "@aws-cdk/aws-ecr": "0.0.0",
     "@aws-cdk/aws-events": "0.0.0",
     "@aws-cdk/aws-iam": "0.0.0",
-    "@aws-cdk/aws-ec2": "0.0.0",
-    "@aws-cdk/aws-secretsmanager": "0.0.0",
+    "@aws-cdk/aws-lambda": "0.0.0",
     "@aws-cdk/aws-s3-assets": "0.0.0",
-    "@aws-cdk/cx-api": "0.0.0"
+    "@aws-cdk/aws-secretsmanager": "0.0.0",
+    "@aws-cdk/aws-sns": "0.0.0",
+    "@aws-cdk/cloud-assembly-schema": "0.0.0",
+    "@aws-cdk/core": "0.0.0",
+    "@aws-cdk/cx-api": "0.0.0",
+    "aws-sdk": "^2.848.0",
+    "constructs": "^3.3.69"
   },
-  "bundledDependencies": [],
+  "bundledDependencies": [
+    "aws-sdk"
```

Comment:
Lambdas already come with the `aws-sdk` 2.x, you don't need this.

Add `aws-sdk` to `devDependencies` (instead of regular dependencies). We don't need to bundle it.

Reply:
ACK how did this slip through